.github/workflows/ubuntu.yml

```diff
@@ -28,6 +28,10 @@ env:
   RUN_SIMPLE_TESTS: ${{ github.event_name == 'push' || inputs.run-tests == 'true' }}
   RUN_EXTENDED_TESTS: ${{ github.event_name == 'schedule' || inputs.run-tests == 'true' }}
   REF: ${{ inputs.target_branch =='' && github.ref || inputs.target_branch}}
+  VCPKG_ROOT: ${{ github.workspace }}/vcpkg
+  vcpkgPackages: yaml-cpp antlr4 boost-test
+  triplet: x64-linux-dynamic
```

Comment:
I think we'd better use static linking, it will be far easier to package.
We already link almost everything statically: ortools, its solvers, minizip (not sirius but I don't see any reason for this).

Reply:
i did this because of boost which seems to be linked dynamically (?) 
will find a way